pkg/kube/kube_handler.go

```diff
@@ -506,6 +506,7 @@ func (a *userHandler) applyAppRepositorySecret(repoSecret *corev1.Secret, reques
 	return err
 }
 
+// Deprecated: Remove when the new Package Repository API implementation is completed
```

Comment:
Great! I'm really looking forward to start removing old code from the repo and, especially, moving stuff from the root `/pkg` folder to the `/kubeapps-apis` directory when possible.

Reply:
Yes! When the Package Repository API is at full speed we can get rid of a lot of legacy code.